build.sh

```diff
@@ -36,10 +36,7 @@ run() {
 
 
 
-run $PAKET_EXE restore
-
-[ ! -e build.fsx ] && run $PAKET_EXE update
-[ ! -e build.fsx ] && run $FAKE_EXE init.fsx
+run $PAKET_EXE restore --group build
```

Comment:
Why restore only the build group ? You will miss others deps no ? Like `dotnet-fable`

Reply:
This is for the top level one. Also seems like paket wasn't restoring all groups for whatever reason.